docs/v0.3/sdk/go/xfgss-faq.md

```diff
@@ -0,0 +1,35 @@
+---
+slug: /xfgss/faq
+---
+
+# FAQ
+
+### What's the difference between Dagger 0.2 and 0.3?
+
+The primary difference is that Dagger 0.3 exposes its functionality via a low-level language-agnostic API framework and provides native language SDKs that communicate with this API. This has two main benefits:
+- It enables CI/CD pipeline authors to use whichever language feels natural and intuitive to them, without having to think about interoperability.
+- Rather than reimplementing the same automation in every possible programming language, it only needs to be implemented once and is then reusable by all. As a result, two cooperating teams no longer necessarily have to come to consensus on a common language.
+
+### What API framework does Dagger 0.3 use?
+
+Dagger 0.3 uses GraphQL as its low-level language-agnostic API framework.
```

Comment:
I've kept mention of GraphQL to a minimum, and only in this FAQ. Please let me know if we need to make this more visible.